qa/workunits/cephadm/test_cephadm.sh

```diff
@@ -160,7 +129,7 @@ function nfs_stop()
     # stop the running nfs server
     local units="nfs-server nfs-kernel-server"
     for unit in $units; do
-        if systemctl status $unit; then
+        if systemctl status $unit < /dev/null; then
```

Comment:
```suggestion
        if systemctl status $unit > /dev/null; then
```

Reply:
I tried that first, but it doesn't prevent systemctl status from launching the pager; ``>`` does

Reply:
maybe `systemctl --no-pager` ?